can-bus/json/FSM/FSM_alerts.json

```diff
@@ -22,9 +21,10 @@
     "faults": [
         "FSM_FAULT_MISSING_HEARTBEAT",
         "FSM_FAULT_STATE_FAULT",
+        "FSM_FAULT_APPS_HAS_DISAGREEMENT",
         "FSM_FAULT_PAPPS_IS_OCSC_IS_ACTIVE",
         "FSM_FAULT_SAPPS_IS_OCSC_IS_ACTIVE",
-        "FSM_FAULT_APPS_HAS_DISAGREEMENT",
-        "FSM_FAULT_FLOW_METER_HAS_UNDERFLOW"
+        "FSM_FAULT_FLOW_METER_HAS_UNDERFLOW",
+        "FSM_FAULT_BRAKE_ACC_DISAGREEMENT"
```

Comment:
Brake and accelerator pedal disagreement has been upgraded to a fault (so if you press accelerator and brake simultaneously, the car will open contactors). This is a rules requirement.

Reply:
man no burnouts? cmon.

Reply:
if it was up to me 😢